src/components/plots/piePlot.ts

```diff
@@ -98,10 +98,10 @@ export module Plots {
       return this;
     }
 
-    public outerRadius(): AccessorScaleBinding<D, number>;
-    public outerRadius(outerRadius: number | _Accessor): Plots.Pie<D>;
-    public outerRadius(outerRadius: D | _Accessor, scale: Scale<D, number>): Plots.Pie<D>;
-    public outerRadius(outerRadius?: number | _Accessor | D, scale?: Scale<D, number>): any {
+    public outerRadius<O>(): AccessorScaleBinding<O, number>;
+    public outerRadius(outerRadius: number | Accessor<number>): Plots.Pie;
+    public outerRadius<O>(outerRadius: O | Accessor<O>, scale: Scale<O, number>): Plots.Pie;
+    public outerRadius<O>(outerRadius?: number | Accessor<number> | O | Accessor<O>, scale?: Scale<O, number>): any {
```

Comment:
Would it make more sense if `innerRadius()` and `outerRadius()` took the same generic type?
```Typescript
```